External/FEXCore/Source/Interface/Core/JIT/Arm64/ALUOps.cpp

```diff
@@ -524,7 +524,7 @@ DEF_OP(LDiv) {
       mov(x1, GetReg<RA_64>(Op->Header.Args[0].ID()));
       mov(x2, GetReg<RA_64>(Op->Header.Args[2].ID()));
 
-#if _M_X86_64
+#if VIXL_SIMULATOR
```

Comment:
These are leftover, can just delete anything you've changed over to `#if VIXL_SIMULATOR`. It isn't expected to work anymore.

Reply:
Removed